layouts/partials/component/frontpage.html

```diff
@@ -10,7 +11,9 @@
 {{ $mobileWidth := .page.Param "river-promo-mobile-width" }}
 {{ $mobileHeight := .page.Param "river-promo-mobile-height" }}
 
-
+{{ with $cta }}
+  {{ partial . }}
```

Comment:
I didn't know you were allowed to do this. Neat. We can change it to `{{ partial . $ }}` down the road if we ever need to pass the page props to the CTA.

Reply:
Great, I've noted this for later